### DISCUSSION TITLE
Running FDS on linux

### DISCUSSION THREAD

**Discussion by FinleyRae8** [firemodels/fds#13484] [General]
Hello, 

I am trying to run a model in linux and face an error I have never seen before. It will do one time step then instantly crash. I have run models with more cells and worked on linux. The model I am running has a fine mesh (0.125) with a 'vent' supplying a custom gas at high velocity and temperatures. I am able to run the models on windows through my laptop, however the linux PCs will do this much faster. The error I get is:

BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
RANK 3 PID 137434 RUNNING AT [computer name]
KILLED BY SIGNAL: 9 (Killed)

If anyone knows how to fix this would be much appreciated!

**Comment by FinleyRae8** [comment]
I figured it out so just incase anyone has the same issue, I just had to increase the limits linux has set as default through the ulimit function.

**Reply by ESH-y** [reply]
hi!
I would appreciate it if you could explain it more please
I have the same error and I'm struggling to figure out how to solve it
thank

**Reply by drjfloyd** [reply]
@ESH-y  If you are running on a Linux system see: https://github.com/firemodels/fds/wiki/Setting-Stack-Size
